deploy-board/deploy_board/webapp/group_view.py

```
@@ -543,7 +543,7 @@ def add_alarms(request, group_name):
         autoscaling_groups_helper.add_alarm(request, group_name, [alarm_info])
     except:
         log.error(traceback.format_exc())
```

Is the exception logged twice in this case? If it does, we don't need a try-catch here.

---

I noticed the pattern in this file. This is probably fine then.

---

It technically logs it twice to the console. If I move if statement and its variables to before the try statement it logs once and still renders the error page, while still maintaining the original functionality.

---

Looks like the try-catch was to log the error, and fail silently. If the purpose of this PR is to not fail silently, then you can simply remove the try-catch. I think that way any exception will be logged and surface up to the user.

---

When you put the `raise` in the catch, it will propagate the exception and thus not fail silently.

---

I see and now that the IF statement is outside of the try-catch, I will remove the `raise` statement that I put in originally. That will allow `autoscaling_groups_helper.add_scheduled_actions()` to fail silently